SPECS/python-lxml/python-lxml.spec

```diff
@@ -69,17 +61,16 @@ rm -rf %{buildroot}
 %files
 %defattr(-,root,root,-)
 %license LICENSES.txt
-%{python2_sitelib}/lxml/*
```

Comment:
If we're essentially removing `python-lxml`, I think we should completely remove this `%files` section. Otherwise, we're going to continue to produce newer versions and releases of this package, which only install the `LICENSES.txt` file. It's going to send a confusing message, that we're still supporting the Python 2 version of the package. Ideally, I'd re-name the directory and the spec file to `python3-lxml` and replace the current `%files` section with what's under `%files -n python3-lxml`.

Reply:
+1
If you remove the `%files` section then the %license should be moved to `%files -n python3-lxml`